listenbrainz/webserver/views/api.py

```diff
@@ -181,7 +181,7 @@ def get_listen_count(user_name):
 
 
 @api_bp.route("/user/<user_name>/playing-now", methods=['GET', 'OPTIONS'])
-@crossdomain()
+@crossdomain(headers="Authorization")
```

Comment:
Other removals look good but this and 1 or 2 other instances of passing Authorization header in the file are present and should also be removed.